### PR TITLE
[swift] Handle string array parameters

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @ohayon</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: fastlane/swift/RubyCommand.swift [hunk 1]
```diff
@@ -62,61 +64,61 @@ struct RubyCommand: RubyCommandable {
...
             jsonParts.append(argsJson)
         }
-        
+
         if let className = className {
             let classNameJson = "\"className\" : \"\(className)\""
             jsonParts.append(classNameJson)
         }
-        
+
```
Comment (taquitos) [inline fastlane/swift/RubyCommand.swift]: so much added white space!

Comment (taquitos) [reply fastlane/swift/RubyCommand.swift]: 👍

**Review by taquitos — APPROVED** [review]
["Amazing", "the best", ":balloon:", ":goat:"]

**Pull request merged**